tools/total-contributions

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import pathlib
+import subprocess
+import sys
+from datetime import datetime
+from typing import Dict, List
+
+
+def add_log(committer_dict: Dict[str, int], input: List[str]) -> None:
+    for dataset in input:
+        committer_name = dataset.split("\t")[1]
+        commit_count = int(dataset.split("\t")[0])
+        if committer_name in committer_dict:
+            committer_dict[committer_name] += commit_count
+        else:
+            committer_dict[committer_name] = commit_count
+
+
+def retrieve_log(repo: str, lower_version: str, upper_version: str) -> List[str]:
+    return subprocess.check_output(
+        ["git", "shortlog", "-s", lower_version + ".." + upper_version],
+        cwd=find_path(repo),
+        text=True,
+    ).splitlines()
+
+
+def find_version(time: str, repository: str) -> str:
+    """Find the latest release version for the target repository as of the
+    specified time.
+    """
+    versions_list = subprocess.check_output(
+        ["git", "tag", "-l"], cwd=find_path(repository), text=True
+    ).splitlines()
+    for version in versions_list:
+        version_time = subprocess.check_output(
+            ["git", "log", "-1", "--format=%ai", version],
```

Comment:
Probably want `%ci` here too

Reply:
Applied already while debugging wrong output caused in part by this (haven't pushed yet)